### PR TITLE
Indicate Node 0.12.x support.

### PR DESCRIPTION
**Pull request by apaprocki**

### CODE REVIEW

**Comment by ljharb** [comment]
Since it's `nvm`, people can also use "node" or "stable" to always refer to the most recent release. Worth mentioning? (Also "iojs" for the latest `io.js` version)

**Comment by apaprocki** [comment]
@ljharb I made it a bit more verbose and clear.

**Comment by ljharb** [comment]
Awesome thanks, LGTM!

File: user/languages/javascript-with-nodejs.md [hunk 1]
```diff
@@ -21,20 +22,26 @@ You can choose Node.js and io.js versions to run your tests by adding the follow
...
 
-* 0.10.x (recent stable release)
+* 0.12.x (recent stable release)
+* 0.11.x (latest development release, may be unstable)
+* 0.10.x
 * 0.8.x
 * 0.6.x
-* 0.11.x (latest development release, may be unstable)
 * iojs (recent stable release of io.js)
+* iojs-v1.0.2
```
Comment (BanzaiMan) [inline user/languages/javascript-with-nodejs.md]: This is not true. As of 2015/02/08, the only version of io.js pre-installed is 1.1.0.

Comment (apaprocki) [inline user/languages/javascript-with-nodejs.md]: I removed text referring to io.js v1.0.2 (it was inherited from the existing docs).

**Comment by BanzaiMan** [comment]
Thanks.

**Pull request merged**